src/frontends/pytorch/src/op/logcumsumexp.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/cum_sum.hpp"
+#include "openvino/op/exp.hpp"
+#include "openvino/op/log.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_logcumsumexp(const NodeContext& context) {
+    // aten::logcumsumexp(Tensor self, int dim) -> Tensor
+    num_inputs_check(context, 2, 2);
+    auto input = context.get_input(0);
+    auto dim = context.get_input(1);
+
+    // First compute exp(input)
+    auto exp = context.mark_node(std::make_shared<v0::Exp>(input));
+    
+    // Then compute cumsum of the exponentials
+    auto cumsum = context.mark_node(std::make_shared<v0::CumSum>(exp, dim));
+    
+    // Finally take log of the result
+    auto log = context.mark_node(std::make_shared<v0::Log>(cumsum));
```

Comment:
we may have exponent explosion for large `input` elements during computation of exp^(input_el).
In order to avoid this, please do the following:
1. compute max elements along the required dim. `max=ReduceMax(input, dim)`
2. `ln(cumsum(exp(input), dim)) = ln(exp(max) cumsum(exp(input-max), dim)) = max + ln(cumsum(exp(input-max), dim))`